cypress/integration/Services/Measure Service/MeasureBundle.spec.ts

```diff
@@ -0,0 +1,382 @@
+                        authorization: 'Bearer ' + accessToken.value
+                    }
+                }).then((response) => {
+                    expect(response.status).to.eql(403)
+
+                })
+            })
+        })
+    })
+
+})
+describe('Measure Bundle end point returns cqlErrors as true', () => {
+    before('Create Measure',() => {
+
+        cy.setAccessTokenCookie()
+        //Create New Measure
+        cy.getCookie('accessToken').then((accessToken) => {
+            cy.request({
+                url: '/api/measure',
+                headers: {
+                    authorization: 'Bearer ' + accessToken.value
+                },
+                method: 'POST',
+                body: {
+                    'measureName': measureName,
+                    'cqlLibraryName': CqlLibraryName,
+                    'model': model,
+                    'measureScoring': measureScoring,
+                    'cql': invalidmeasureCQL,
+                    'measurementPeriodStart': mpStartDate + "T00:00:00.000Z",
+                    'measurementPeriodEnd': mpEndDate + "T00:00:00.000Z"
+                }
+            }).then((response) => {
+                expect(response.status).to.eql(201)
+                expect(response.body.id).to.be.exist
+                cy.writeFile('cypress/fixtures/measureId', response.body.id)
+            })
+        })
+
+        //create Measure Group
+        cy.getCookie('accessToken').then((accessToken) => {
+            cy.readFile('cypress/fixtures/measureId').should('exist').then((retrievedMeasureID) => {
+                cy.request({
+                    url: '/api/measures/' + retrievedMeasureID + '/groups/',
+                    method: 'POST',
+                    headers: {
+                        authorization: 'Bearer ' + accessToken.value
+                    },
+                    body: {
+                        "scoring": measureScoring,
+                        "population": 
+                        {
+                            "initialPopulation": PopIniPop,
+                            "denominator": PopDenom,
+                            "denominatorExclusion": PopDenex,
+                            "denominatorException": PopDenexcep,
+                            "numerator": PopNum,
+                            "numeratorExclusion": PopNumex
+                        }
+                    }
+                }).then((response) => {
+                        expect(response.status).to.eql(201)
+                        expect(response.body.id).to.be.exist
+                        cy.writeFile('cypress/fixtures/groupId', response.body.id)
+                })
+            })
+        })
+    })
+
+    beforeEach('Set Access Token',() => {
+
+        OktaLogin.Login(true)
+
+    })
+
+    after('Clean up',() => {
+        Utilities.deleteMeasure(measureName, CqlLibraryName, measureScoring)
+
+    })
+    it('Log into the UI and save Measure CQL', () => {
```

Comment:
not sure I understand this test? Are we testing the UI or the service? I think it might be best for the service tests to not rely on any UI at all.

Reply:
yea -- this change is a little odd because the new cqlErrors flag is only going to be updated once the Measure CQL is saved from the front end

Reply:
I have moved the tests that are related to that cqlErrors flag (which can only be updated by physically clicking on the save button for Measure CQL) to our UI tests